styles/swiper.css

```
@@ -0,0 +1,44 @@
+.swiper {
+  --swiper-navigation-size: 24px;
+  position: relative;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)